src/Whoops/Resources/views/header.html.php

```diff
@@ -17,11 +17,16 @@
     COPY
   </button>
 
-  <p class="exc-message">
+  <div class="exc-message">
     <?php if (!empty($message)): ?>
       <?php echo $tpl->escape($message) ?>
     <?php else: ?>
       <span class="exc-message-empty-notice">No message</span>
     <?php endif ?>
-  </p>
+
+    <ul class="icon-list">
```

Comment:
Please use a more descriptive class name. `search-for-help` seems more suitable.